package.json

```diff
@@ -232,5 +232,13 @@
     "safari5": [
       "Safari < 5"
     ]
+  },
+  "dependencies": {
+    "opencollective-postinstall": "^2.0.2",
+    "opencollective": "^1.0.3"
+  },
+  "collective": {
+    "type": "opencollective",
+    "url": "https://opencollective.com/cssnano"
   }
-}
+}
```

Comment:
We need move this stuff in `packages/cssnano/package.json`

Reply:
@evilebottnawi I think it would be better if this is moved to default preset package. 
As it will automatically come in cssnano and also, those who are using default-preset without cssnano can see the message as well